app/views/pages/privacy_policy.html.erb

```diff
@@ -22,7 +22,7 @@
       <li><%= t('.list_item2') %></li>
     </ul>
 
-    <br>
+    <br/>
```

Comment:
Picky - but closing a `<br>` tag in HTML 5 [is optional](http://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br) - with the default being not to.